mkdocs.yml

```diff
@@ -31,6 +31,7 @@ nav:
   - Overview: index.md
   - Release Process: release-process.md
   - Security: SECURITY.md
+  - Upgrading: upgrading.md
```

Comment:
Lets move this below `Releases`. 
https://github.com/argoproj-labs/argocd-operator/blob/cba8d10a68358330b7e627ab81d5c498ded8d21e/mkdocs.yml#L78

We can merge this PR. You can handle this change in  your other PR.